src/stylesheets/components/_example.scss

```diff
@@ -0,0 +1,177 @@
+// Example styles
+.app-c-example {
+  position: relative;
+  margin: $govuk-gutter 0;
+  border: 1px solid $govuk-border-colour;
+  background: $govuk-grey-3;
+}
+
+.app-c-example--tabs {
+  margin-bottom: 0;
+  @include mq($from: desktop) {
+    margin-bottom: -1px;
+  }
+}
+
+.app-c-example__new-window {
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 5px $govuk-gutter / 2;
+  @include govuk-core-14;
+  background: $govuk-grey-3;
+
+  a {
+    color: $govuk-link-colour;
+    text-decoration: none;
+  }
+
+  &:before {
+    @include govuk-core-14;
+    content: "EXAMPLE";
+    margin-right: 10px;
+  }
+}
+
+.app-c-example__frame {
+  display: block;
+  width: 100%;
+  max-width: 100%;
+  border: 0;
+  border-right: 1px solid $govuk-border-colour;
+  background: $govuk-white;
+}
+
+.app-c-example__frame--resizable {
+  overflow: auto;
+  resize: both;
+}
+
+// Tabs styles
+.app-c-tabs {
+  margin: -1px auto;
+  padding: 0;
+  overflow: visible;
+  list-style-type: none;
+  border: 1px solid $govuk-border-colour;
+
+  @include mq($until: desktop) {
+    display: none;
+  }
+}
+
+.app-c-tabs__item {
+  display: inline-block;
+  position: relative;
+  z-index: 2;
+  margin: 0;
+
+  a {
+    display: block;
+    margin-top: -1px;
+    padding: $govuk-gutter-half;
+    border-top: 2px solid transparent;
+    border-bottom: 2px solid transparent;
+    color: $govuk-link-colour;
+    background: inherit;
+    font-weight: bold;
+    text-decoration: none;
+  }
+}
+
+.app-c-tabs__item--current {
+  a {
+    border-top-color: $govuk-blue;
+    border-bottom-color: $govuk-blue;
+    background-color: darken($govuk-grey-4, 3%);
+  }
+}
+
+// Tab drawer heading for accordion look on mobile
+.app-c-tabs__heading {
+  a {
+    display: none;
+    position: relative;
+    z-index: 2;
+    padding: $govuk-gutter-half;
+    border: 1px solid $govuk-border-colour;
+    border-top: 0;
+    color: $govuk-link-colour;
+    background: inherit;
+    font-weight: bold;
+    text-decoration: none;
+
+    &:visited {
+      color: $govuk-link-colour;
+    }
+
+    @include mq($until: desktop) {
+      display: block;
+    }
+  }
+}
+
+.app-c-tabs__heading--current {
+  background-color: darken($govuk-grey-4, 3%);
+}
+
+// Tab containers
+.app-c-tabs__container {
+  position: relative;
+  margin-top: -2px;
+  border: 1px solid $govuk-border-colour;
+
+  @include mq($until: desktop) {
+    border-top: 0;
+  }
+}
+
+.app-c-tabs__container pre {
+  padding-bottom: $govuk-gutter;
+  border: 0;
+}
+
+// Close container button
+.app-c-link--close {
+  position: absolute;
+  right: $govuk-gutter-half;
+  bottom: $govuk-gutter-half / 2;
+  text-decoration: none;
+}
+
+// Generic chevron
+// can be rotated for --right --bottom --left versions
+.app-u-chevron--top::before {
```

Comment:
Likewise, I don't think we should be putting utilities / helpers like this inside component files.